test/Test/Management.hs

```diff
@@ -1,111 +1,133 @@
-import Michelson.Runtime (ExecutorError)
+import Lorentz
+import Lorentz.Contracts.BaseDAO (mkStorage)
+import Morley.Nettest
 import Test.Common
 import Util.Named
 
-testOwner, wallet1, wallet2 :: Address
-testOwner = genesisAddress1
-wallet1 = genesisAddress2
-wallet2 = genesisAddress3
-
 -- | We test non-token entrypoints of the BaseDAO contract here
 test_baseDAO :: [TestTree]
 test_baseDAO =
   [ testGroup "Ownership transfer"
-    [ testCase "Transfer ownership entrypoint authenticates sender" $
-        integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
-          err <- expectError $ withSender wallet1 $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
-          expectNotAdmin err
+    [ testCase "transfer ownership entrypoint authenticates sender" $
+        nettestTestExpectation $ uncapsNettest $ do
+          withOriginated 2 (\(owner:_) -> initialStorage owner) $ \[_, wallet1] baseDao ->
+            callFrom (AddressResolved wallet1) baseDao (Call @"Transfer_ownership") (#newOwner .! wallet1)
+              & expectNotAdmin
 
     , testCase "sets pending owner" $
-        integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
-          withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
-          lExpectStorage baseDAO (checkPendingOwner (Just wallet1))
+        nettestTestExpectation $ uncapsNettest $ do
+          withOriginated 2 (\(owner:_) -> initialStorage owner) $
+            \[owner, wallet1] baseDao -> do
+                callFrom (AddressResolved owner) baseDao (Call @"Transfer_ownership")
+                  (#newOwner .! wallet1)
+                callFrom (AddressResolved wallet1) baseDao (Call @"Accept_ownership") ()
 
     , testCase "does not set administrator" $
-        integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
-          withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
-          lExpectStorage baseDAO (checkAdministrator testOwner)
-
-    , testCase "rewrites existing pending owner" $
-        integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
-          withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
-          lExpectStorage baseDAO (checkPendingOwner (Just wallet1))
-          withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet2)
-          lExpectStorage baseDAO (checkPendingOwner (Just wallet2))
+        nettestTestExpectation $ uncapsNettest $ do
+          withOriginated 2 (\(owner:_) -> initialStorage owner) $
+            \[owner, wallet1] baseDao -> do
+              callFrom (AddressResolved owner) baseDao (Call @"Transfer_ownership")
+                (#newOwner .! wallet1)
+              -- Make the call once again to make sure the admin still retains admin
+              -- privileges
```

Comment:
Previously you were checking the storage using `lExpectStorage`. I think this approach sort of works as well, but did you consider using `getStorage` or `checkStorage` available in `Morley.Nettest.Caps`?

Reply:
I didn't use them because we never seem to have used in any of our nettests. 

Some days back, I had asked Diogo about it and he said that we don't usually do storage checkes because storage checks only works for storages that don't have a big map in them. Not sure if that has changed though..

Reply:
Hm, right, it only works for storage without big_maps and can't be used in our case. Probably we'll need this functionality at some point, so we should prioritize https://gitlab.com/morley-framework/morley/-/issues/346. Until it's done, if we ever need to inspect storage, we'll have to stick to Integrational interface in such case.